firebase-vscode/common/messaging/protocol.ts

```diff
@@ -117,6 +131,26 @@ export interface ExtensionToWebviewParamsMap {
    */
   notifyFirebaseConfig: { firebaseJson: FirebaseConfig, firebaseRC: FirebaseRC };
 
+  notifyEmulatorsStopped: {};
+  notifyRunningEmulatorInfo: RunningEmulatorInfo ;
+  notifyEmulatorImportFolder: { folder: string };
 }
 
 export type MessageParamsMap = WebviewToExtensionParamsMap | ExtensionToWebviewParamsMap;
+
+/**
+ * Info to display in the UI while the emulators are running
+ */
+export interface RunningEmulatorInfo {
```

Comment:
There's a `types.d.ts` file adjacent to this one, these interfaces are probably a better fit there (that's where Message lives, and ChannelWithId which seems on par with these - info sent thru messages).

Reply:
Done